src/index.html

```diff
@@ -5,9 +5,212 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="icon" href="./images/favicon.png">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Ubuntu:ital,wght@0,300;0,400;0,500;1,400&display=swap"
+      rel="stylesheet">
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header">
+      <div class="header__top">
+        <div class="logo header__logo"></div>
+        <div class="header__phone"><a class="header__phone_number" href="tel:+1 234 555-55-55">+1 234 555-55-55</a></div>
```

Comment:
```suggestion
header__phone_number
```
можификатор не может быить один